applications/ConstitutiveLawsApplication/custom_processes/advance_in_time_high_cycle_fatigue_process.cpp

```diff
@@ -123,9 +206,75 @@ void AdvanceInTimeHighCycleFatigueProcess::CyclePeriodPerIntegrationPoint(bool&
 /***********************************************************************************/
 /***********************************************************************************/
 
-void AdvanceInTimeHighCycleFatigueProcess::StableConditionForAdvancingStrategy(bool& rAdvancingStrategy, bool DamageIndicator)
+void AdvanceInTimeHighCycleFatigueProcess::NoLinearitiesInitiationAndAccumulation(double& rMaximumDamageIncrement, double& rMaximumPlasticDissipationIncrement)
+{
+    auto& process_info = mrModelPart.GetProcessInfo();
+    std::vector<double> damage;
+    std::vector<double> previous_cycle_damage;
+    std::vector<double> plastic_dissipation;
+    std::vector<double> previous_cycle_plastic_dissipation;
+    std::vector<double> plastic_damage;

```

Comment:
damage and dissipation are ad hoc internal variables. Maybe we could think how to name them in a general form. TODO in the future